CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+## v2.2.1
+
+* Improvements
+  * Bump C compiler options to `-O2` from `-Os`. This provides a small, but
+    measurable performance improvement (500ms at boot in a trivial project).
```

Comment:
The 500ms boot performance improvement was on an RPi Zero. I'm not sure what the effect is on the BBB. Probably safer to qualify that the performance improvement was measured elsewhere.